DuckDuckGo/DuckPlayer/YouTubePlayerNavigationHandler.swift

```diff
@@ -137,29 +194,42 @@ extension YoutubePlayerNavigationHandler: DuckNavigationHandling {
         if let url = navigationAction.request.url,
             let (videoID, timestamp) = url.youtubeVideoParams,
             duckPlayer.settings.mode == .disabled {
+            os_log("DP: is Disabled. We should load original video for %s", log: .duckPlayerLog, type: .debug)
             webView.load(URLRequest(url: URL.youtube(videoID, timestamp: timestamp)))
             completion(.allow)
             return
         }
         
         completion(.allow)
-        
     }
     
     // Handle URL changes not triggered via Omnibar
     // such as changes triggered via JS
     @MainActor
     func handleURLChange(url: URL?, webView: WKWebView) {
         
+        // If trying to load the same video while DP is visible
+        // Just open it in Youtube
+        if let url = url,
```

Comment:
DP can now be temporarily disabled for specific cases (Such as Watch in Youtube) scenarios. The basic logic is that if the video you're about to watch is the same video you're already watching, it should open in Youtube and not Duckplayer.